### PR TITLE
Support BitmapText resolution on high-density displays (fixes #5171)

### LINKED ISSUE #5171: BitmapText maps incorrectly on UHD/retina displays
### Expected Behavior
Bitmap text readable on UHD / Retina displays

### Current Behavior
Bitmap text mapped incorrectly, displayed in jumbled fragments of letter forms
bitmaptextuhd
(text still maps correctly on standard HD/1080 PC screens, and renders correctly on all resolutions in Pixi 3)

### Steps to Reproduce
1. Create XML & PNG bitmap font in BMFont
2. Load *.fnt file into Pixijs 4.8.2 project
3. Create new BitmapText object using fnt "face" attribute as font name
4. AddChild to stage

- `pixi.js` version: 4.8.2
- Browser: Chrome 69, Edge 42, Firefox 62
- Device: Surface Pro 3, MacBook Pro

### PR DESCRIPTION
**Pull request by SukantPal**
#5171

### CODE REVIEW

**Comment by SukantPal** [comment]
@bigtimebuddy @ivanpopelyshev This seems to fix this. IDK if this is right fix tho.

**Comment by ivanpopelyshev** [comment]
Is it the only issue about BitmapText resolution? maybe there were others that needed `settings.RESOLUTION` there

**Comment by SukantPal** [comment]
@ivanpopelyshev What do you mean?

File: packages/text-bitmap/src/BitmapFont.ts [hunk 1]
```diff
@@ -41,7 +40,7 @@ export class BitmapFont
         const [info] = data.info;
         const [common] = data.common;
         const [page] = data.page;
-        const res = getResolutionOfUrl(page.file, settings.RESOLUTION);
+        const res = getResolutionOfUrl(page.file, 1);
```
Comment (bigtimebuddy) [inline packages/text-bitmap/src/BitmapFont.ts:43]:
```suggestion
        const res = getResolutionOfUrl(page.file);
```

Comment (bigtimebuddy) [reply packages/text-bitmap/src/BitmapFont.ts:43]: We can actually remove the default value. That's what's set in the function.

**Pull request merged**